### DISCUSSION TITLE
Unable to create stack on AWS account with a CNAME error on (unused) domain

### DISCUSSION THREAD

**Discussion by misslivirose** [mozilla/hubs#3170] [Hubs Cloud Questions] [answered]
Asking for me - 

I'm trying to deploy a stack to my AWS account. Potentially of note, this account already has another AWS stack on it (but the other stack has a completely separate set of domains associated with it). 

When I try to create my stack, it fails to deploy and rolls back at this stage: 
![image](https://user-images.githubusercontent.com/4313320/96025210-acb57600-0e09-11eb-940f-668c2a4761fb.png)

However, I have confirmed that: 
* None of the DNS records for the domain names that I'm using point to anything else
* The CloudFront manager does not show any of the domain names that I'm using as already being in use

Is setting up multiple stacks (both are Hubs Cloud Personal) on a single AWS account supported? Any advice on what to do to fix this error? Should I delete the other stack?

**Comment by misslivirose** [comment]
I deleted my other stack and the error still occurs, as an additional data point.

**Comment by robinkwilson** [comment]
It's possible a Rollback failed to delete the CNAMES from the last deploy on your domains and is failing to make the new stack. I personally haven't had trouble with multiple stack deploys on my account. 

Try **Solution 4 DNS CNAME issues** on Hubs Docs: [AWS Troubleshooting](https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#potential-solutions). Essentially delete leftover CNAMES on your domains you're looking to deploy to. NOTE: if you have another service that depends on a specific CNAME, do not delete it. (We should make this a separate error on the AWS Troubleshooting page definitely)

**Reply by misslivirose** [reply]
Thanks! I did this and still had the issue, but I'll try again. There are two DNS records on each domain that I can't edit that point to the AWS name servers. 

I'll also note that deleting the stacks does not seem to clean up the DNS records correctly, but I'm not sure if that's supposed to happen or not.

**Comment by misslivirose** [comment, marked as answer]
I deleted the DNS records and then tried (as a work around) swapping the domains I was using for my internal and short link records. The stack successfully completed that time.
